src/applications/proxy-rewrite/tests/redirects/index.unit.spec.js

```diff
@@ -64,15 +64,25 @@ describe('Redirect replaced pages', () => {
 });
 
 describe('Validate crossDomainRedirects.json', () => {
-  const redirectsBySource = redirects.reduce((grouped, redirect) => {
-    const fullPath = `https://${redirect.domain}${redirect.src}`;
-    const items = grouped[fullPath] || [];
-    return {
-      ...grouped,
-      [fullPath]: items.concat(redirect.dest),
-    };
-  }, {});
+  const nonSubdomainRedirects = redirects.reduce((redirectArray, current) => {
```

Comment:
I don't want to hold up this PR, but think for clarity and succinctness it would be to use the Array.prototype.filter function, since we're just building a new based on a boolean predicate.

 It would be:
  `const nonSubdomainRedirects = redirects.filter((redirect) => !redirect.isToSubdomain)`

Reply:
good call, just updated